examples/libp2p-in-the-browser/package.json

```diff
@@ -8,6 +8,7 @@
   ],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "parcel build index.html",
```

Comment:
probably not needed with the new setup

Reply:
Just tried this out, but we need it otherwise the tests do not run.